initial-config/static/globals.json

```diff
@@ -44,6 +44,12 @@
         "secretKey": "reCaptchaSecretKey",
         "websiteKey": "reCaptchaWebsiteKey"
     },
+    "kongAdapter": {
```

Comment:
Like above, these things should be added in an update step, not only to the initial configuration. Otherwise a user will never see these settings if he created the configuration prior this feature being released.

Reply:
Done